packages/features/sales-escrows/modals/ConfirmSalesActionModal.tsx

```diff
@@ -62,6 +67,7 @@ export const ConfirmSalesActionModal = ({
               horizontal: 'right',
             },
           });
+          setInProcess?.(false);
```

Comment:
Rather than calling this in multiple places, it's better to wrap everything in a try/catch/finally statement, set it to true at the top of the try, and set it to false in the finally block. This ensures it is set to false in every scenario, even if there is an error.